pkg/api/types.go

```diff
@@ -2187,6 +2189,8 @@ type OwnerReference struct {
 	// UID of the referent.
 	// More info: http://releases.k8s.io/HEAD/docs/user-guide/identifiers.md#uids
 	UID types.UID `json:"uid"`
+	// If set to true this reference points to managing controller
+	Controller bool `json:"controller"`
```

Comment:
We should make this optional, defaulting to false. Otherwise, users would always need to explicitly set it to false, which would be annoying.

Please add omitempty and make it a pointer, so that it works with kubectl apply.